repos/system_upgrade/el7toel8/actors/repositoriesblacklist/libraries/repositoriesblacklist.py

```diff
@@ -2,12 +2,53 @@
 from leapp.libraries.common.config import get_product_type
 from leapp.libraries.stdlib import api
 from leapp.models import (
+    CustomTargetRepository,
     RepositoriesBlacklisted,
     RepositoriesFacts,
     RepositoriesMap,
 )
 
 
+def _report_using_unsupported_repos(repos):
+    report = [
+        reporting.Title("Using repository not supported by Red Hat"),
+        reporting.Summary(
+            "The following repositories have been used for the "
+            "upgrade, but they are not supported by the Red Hat.:\n"
+            "- {}".format("\n - ".join(repos))
+        ),
+        reporting.Severity(reporting.Severity.HIGH),
+        reporting.Tags([reporting.Tags.REPOSITORY]),
+    ]
+    reporting.create_report(report)
+
+
+def _report_excluded_repos(repos):
+    api.current_logger().info(
+        "The optional repository is not enabled. Excluding %r "
+        "from the upgrade",
+        repos,
+    )
+
+    report = [
+        reporting.Title("Excluded RHEL 8 repositories"),
+        reporting.Summary(
+            "The following repositories are not supported by "
+            "Red Hat and are excluded from the list of repositories "
+            "used during the upgrade.\n- {}".format("\n- ".join(repos))
+        ),
+        reporting.Severity(reporting.Severity.INFO),
+        reporting.Tags([reporting.Tags.REPOSITORY]),
+        reporting.Flags([reporting.Flags.FAILURE]),
+        reporting.Remediation(
+            hint="Use leapp upgrade with the following options: {}.".format(
+                " ".join(["--enablerepo " + repo for repo in repos])
```

Comment:
It's no clear why user should use the option. Reading that, I believe many users could take it as "they have to run leapp with the option". As well, we do not want to use concatenation of string in the code so much. Use rather the format please.

```suggestion
            hint="If you still require to use the excluded repositories during the upgrade, execute leapp with the following options: {}.".format(
                " ".join(["--enablerepo {}".format(repo) for repo in repos])
```